webhooks-extension/tekton/resources.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   - name: url
     value: https://github.com/tektoncd/experimental 
   - name: revision
-    value: INSERT_COMMIT_ID_TO_PUBLISH_HERE
```

Comment:
would it not be better to have this so it'll fail rather than potentially publishing the wrong thing?

Reply:
Yeah good idea, I've gone with `INSERT_TAG_HERE`